labonneboite/common/search.py

```diff
@@ -786,7 +787,8 @@ def get_offices_from_es_and_db(json_body, sort, rome_codes, hiring_type):
         sorting.SORT_FILTER_SCORE: 3,  # (boosted_romes_sort, randomized_score_sort, distance_sort)
     }[sort]
     # Check if this is a request with long/lat
-    has_geo_distance = next((dic for dic in json_body.get('sort') if '_geo_distance' in dic), None) != None
+    first_dic_with_geo_distance = next((dic for dic in json_body.get('sort') if '_geo_distance' in dic), None)
```

Comment:
Ok en cherchant un peu comment renvoyer le premier élément d'une liste ou `None` si la liste est vide je suis tombé sur ta méthode:

https://stackoverflow.com/questions/363944/python-idiom-to-return-first-item-or-none
![image](https://user-images.githubusercontent.com/10533583/75021776-9be07a80-549d-11ea-84c4-2d4efa847f07.png)

Mais je reste confus. Par exemple qu'est ce qui manquerait à cette version plus simple?

`has_geo_distance = len([d for d in json_body.get('sort') if '_geo_distance' in d]) > 0`

Reply:
Encore mieux:

`has_geo_distance = any([('_geo_distance' in d) for d in json_body.get('sort')])`